lib/workers/repository/onboarding/branch/check.ts

```diff
@@ -129,6 +129,6 @@ export const isOnboarded = async (config: RenovateConfig): Promise<boolean> => {
   throw new Error(REPOSITORY_CLOSED_ONBOARDING);
 };
 
-export const onboardingPrExists = async (
+export const getOnboardingPr = async (
```

Comment:
should be refactored in separate PR and converted to regular function.

Reply:
- https://github.com/renovatebot/renovate/pull/17965